Makefile

```diff
@@ -7,8 +7,13 @@ else
     CFLAGS += -O2
 endif
 
-mkmimo: mkmimo.o
+ifdef MULTITHREADED
```

Comment:
I changed `MULTITHREADED` to be the flag we explicitly had to specify, so that I could make sure tests were still working in the nonblocking version after we refactored the code.